.travis.yml

```diff
@@ -30,5 +30,6 @@ script:
   - node -e "const Cesium = require('./');"
   - NODE_ENV=development node Specs/test.cjs
   - NODE_ENV=production node Specs/test.cjs
+  - node Specs/test.mjs
```

Comment:
Do we need to test this in both `production` and `development` mode too, like we do with `test.cjs`? Looking at `test.cjs`, it is not obvious to me what the point of setting the environment to `production` is.

Reply:
Good question! So `test.cjs` imports from the root `cesium` module. In out `package.json`, we've defined [conditional exports](https://nodejs.org/api/packages.html#conditional-exports) such that requiring or importing `cesium` which grab a different file depending on the environment. In CJS with a require, we redirect to `index.cjs`, which in turn requires either the minified release or unminified release depending on whether or not the environment is production. (We've recently had [some discussion whether or not `index.cjs` and the runtime operations are needed or not](https://github.com/CesiumGS/cesium/issues/9212#issuecomment-1218449631).)

MJS on the other hand can handle ESM directly, so imports from `Source/Cesium.js` file. We could (and probably should) have `test.mjs` import from the root `cesium`, and it should be functionally equivalent to `Source/Cesium.js`, and so would also verifies our `package.json` configuration as well.